test/faker/tv_shows/test_aqua_teen_hunger_force.rb

```diff
@@ -10,4 +10,8 @@ def setup
   def test_character
     assert @tester.character.match(/\w/)
   end
+
+  def test_quote
+    assert @tester.character.match(/\w/)
```

Comment:
Looking at other test cases I think this line should be

```suggestion
    assert @tester.quote.match(/\w/)
```